### DISCUSSION TITLE
Nether Reactor Removed from editor

### DISCUSSION THREAD

**Discussion by hexagonstudios0** [Mojang/minecraft-editor#99] [Feedback]
The nether reactor core was removed from Editor, but the glowing obsidian block is in the Editor still.

The nether reactor core was in the Editor in version 1.19.80.20
![image](https://user-images.githubusercontent.com/103085649/227725620-c32cd2fe-5207-4b59-bc5d-11d0b879eb5e.png)

CLOSED

Solution: [][(url)](https://www.amuletmc.com/)

**Comment by minecraftchloe** [comment]
As designed!
